packages/cosmic-proto/package.json

```
@@ -24,21 +24,17 @@
       "default": "./dist/vatsafe.js"
     },
     "./package.json": "./package.json",
-    "./cosmos": {
-      "types": "./dist/codegen/cosmos/bundle.d.ts",
-      "default": "./dist/codegen/cosmos/bundle.js"
+    "./cosmos/*.js": {
```

I'd like to expose all the `./dist/codegen/agoric` files under `@agoric/cosmic-proto/agoric/*` for consistency with other chains, as well as the (backward compatible) module-specific export paths stripped of their `agoric/` prefix.

```suggestion
    "./agoric/*.js": {
      "types": "./dist/codegen/agoric/*.d.ts",
      "default": "./dist/codegen/agoric/*.js"
    },
    "./cosmos/*.js": {
```